migrations/V144__Create_Intake_Request_State_and_Step.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TYPE system_intake_step AS ENUM (
+  'INITIAL_REQUEST_FORM',
+  'DRAFT_BUSINESS_CASE',
+  'GRT_MEETING',
+  'GRB_MEETING',
+  'FINAL_BUSINESS_CASE',
+  'DECISION_AND_NEXT_STEPS'
+);
+
+CREATE TYPE system_intake_state AS ENUM (
+  'OPEN',
+  'CLOSED'
+);
+
+ALTER TABLE system_intakes ADD COLUMN step system_intake_step;
```

Comment:
Spreading each column's creation across three separate statements won't work if there's already data in the table (i.e. in the impl and prod environments); setting the default won't help once the column's already been created and populated with `NULL` for each row, and then setting the column to `NOT NULL` will cause an error. You can test this by temporarily removing this migrations file, bringing the backend up, running `scripts/dev db:seed`, then trying to run this SQL manually; it'll error out, even if you wrap the whole thing in a transaction.

Fortunately, we can add each column with a single statement:
```
ALTER TABLE system_intakes ADD COLUMN step system_intake_step NOT NULL DEFAULT 'INITIAL_REQUEST_FORM';
ALTER TABLE system_intakes ADD COLUMN state system_intake_state NOT NULL DEFAULT 'OPEN';
```

Reply:
Also to note, you can add both columns in one alter TABLE statement if you desire

```sql
ALTER TABLE system_intakes
 ADD COLUMN step system_intake_step NOT NULL DEFAULT 'INITIAL_REQUEST_FORM',
 ADD COLUMN state system_intake_state NOT NULL DEFAULT 'OPEN';

```